decentraland/App.js

```diff
@@ -6,6 +6,8 @@ import AppNavigator from "./AppNavigator";
 import { Provider } from "react-redux";
 import { createStore } from "redux";
 import decentralandApp from "./redux/reducers";
+import { setAccount } from "./redux/actions";
+import { recoverAccount } from "./screens/helpers";
```

Comment:
Now that we're importing this in `App.js` it feels a little weird to get it from `screens/helpers`.

Reply:
Should we move to a `@helpers` subpackage?

Reply:
Something that we would publish to npm or just a dir with a package.json with a name so we can reference it like `@helpers`?

Are you picturing it would be outside of the decentraland app dir to share with other apps or inside it?

Reply:
For the purposes of this PR, something as simple as moving the helpers file outside of the `screens` dir would work.

Reply:
I meant just a dir with a package.json.
Ok, I've moved all functions from `screens/helpers.js` to `helpers.js`

Reply:
Oh, Wait... I've found a bug here.

Reply:
> Oh, Wait... I've found a bug here.

Okay, noted!

Reply:
> I meant just a dir with a package.json.
> Ok, I've moved all functions from `screens/helpers.js` to `helpers.js`

Sounds good, thanks

Reply:
I've merged both helpers, the `starter.js` script uses a helper function and because of that I installed `babel-node` since the `helpers` is using ES6 imports. But it's throwing error because `babel` isn't transpiling the `expo` `SecureStore` import. To fix that quickly I've copied the function from `helper` to `starter`.
I can extract the `checkBlockchain` function to another helper file and use it (to avoid copied code) but it'll add one more file to the root dir. I think that the best solution could be create the "package helpers dir" that will have a `index.js` and `startHelpers.js`, what do you think?

Reply:
That all makes sense. Your last suggestion sounds great! But that can come in a follow-up PR.

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/229

Reply:
Thanks! Resolving this